ee/cron/chunksHostLoads.js

```diff
@@ -0,0 +1,189 @@
+// @ts-check
+const AWS = require('aws-sdk');
+const { callbackify } = require('util');
+
+const config = require('../../lib/config');
+
+// These are used as IDs when reading CloudWatch metrics. They must start with
+// a lowercase letter and contain only numbers, letters, and underscores.
+const PAGE_VIEWS_PER_SECOND = 'pageViewsPerSecond';
+const ACTIVE_WORKERS_PER_SECOND = 'activeWorkersPerSecond';
+const LOAD_BALANCER_REQUESTS_PER_MINUTE = 'loadBalancerRequestsPerMinute';
+
+/**
+ * Finds the maximum value in the given array.
+ * @param {number[]} arr
+ */
+function arrayMax(arr) {
```

Comment:
I generally use [`_.max()`](https://lodash.com/docs/4.17.15#max).

Reply:
Changed in 2f3c116.